File: JJB/1202/합성수_찾기.py
def is_composite(n):
    number = 0
    for i in range(1, n+1):
        if n%i==0:
            number += 1
        if number >= 3:
            return True
    return False

def solution(n):
    answer = 0
    
    for i in range(2, n+1):
        answer += is_composite(i)
    
    return answer

File: JJB/Programmers/231202/합성수_찾기.cpp
#include <string>
#include <vector>
#include <cmath>

using namespace std;

bool is_composite(const int &n)
{
    int number = 0;
    for(int i = 1; i <= n; i++)
    {
        if(n%i==0)
            number++;
        if(number>=3)
            return true;
    }
    return false;
}

int solution(int n) 
{
    int answer = 0;
    
    for(int i = 2; i <= n; i++)
        answer += is_composite(i);
    
    return answer;
}